### PR TITLE
Move Skia gold client to shared location

### PR DESCRIPTION
**Pull request by blasten**
This allows the Android tests to also use the Skia Gold client on LUCI.

### CODE REVIEW

File: testing/skia_gold_client/lib/skia_gold_client.dart [hunk 1]
```diff
@@ -319,21 +358,6 @@ class SkiaGoldClient {
     // image size because those wrong pixels are constrained by
     // `pixelDeltaThreshold` below.
     final int maxDifferentPixels = (screenshotSize * kMaxDifferentPixelsRate).toInt();
```
Comment (mdebbar) [inline testing/skia_gold_client/lib/skia_gold_client.dart:360]: You may want to parametrize this one too. It controls how many pixels in the new image are allowed to differ from the golden image.

Right now, this is hardcoded at 10% but you may want to set a different value on other platforms.

**Review by mdebbar — APPROVED** [review]
Looks good! Just a couple minor comments.

File: lib/web_ui/pubspec.yaml [hunk 2]
```diff
@@ -8,6 +8,8 @@ environment:
 dependencies:
   js: 0.6.4
   meta: 1.3.0
+  skia_gold_client:
+    path: ../../testing/skia_gold_client
```
[the remark this answers is not in the document]
Comment (blasten) [reply lib/web_ui/pubspec.yaml]: done

File: testing/skia_gold_client/lib/skia_gold_client.dart (hunk 1 above)
Comment (blasten) [reply testing/skia_gold_client/lib/skia_gold_client.dart]: 👍 done

**Comment by blasten** [comment]
cc @jmagman in case this is also useful for iOS

**Comment by Hixie** [comment]
test-exempt: refactor with no semantic change

**Pull request merged**